dmscripts/notify_buyers_when_requirements_close.py

```diff
@@ -79,18 +76,11 @@ def main(data_api_url, data_api_access_token, email_api_key, date_closed, dry_ru
 
     logger.info("Notifying users about {briefs_count} closed briefs", extra={'briefs_count': len(closed_briefs)})
 
-    notified_briefs = get_notified_briefs(email_api_key, date_closed)
-
-    logger.info('Brief notifications sent since {date_closed}: {notified_briefs_count}',
-                extra={'date_closed': date_closed, 'notified_briefs_count': len(notified_briefs)})
-
     for brief in closed_briefs:
-        if brief['id'] in notified_briefs:
-            logger.info('Brief notification already sent for brief ID: {brief_id}', extra={'brief_id': brief['id']})
-        elif dry_run:
+        if dry_run:
```

Comment:
Not for this PR, but at some point we should refactor this script to move the `dry-run` flag to the point of sending each email, so we get to the last possible point before doing so, if you see what I mean. This probably needs to be done as part of a wider scripts repo tidy-up though.